site/_guides/global-rate-limiting.md

```diff
@@ -0,0 +1,427 @@
+---
+title: Global Rate Limiting
+layout: page
+---
+
+Starting in version 1.13, Contour supports [Envoy global rate limiting][1].
+In global rate limiting, Envoy communicates with an external Rate Limit Service (RLS) over gRPC to make rate limit decisions for each request.
+Envoy is configured to produce 1+ descriptors for incoming requests, containing things like the client IP, header values, and more.
+Envoy sends descriptors to the RLS, and the RLS returns a rate limiting decision to Envoy based on the descriptors and the RLS's configured rate limits.
+
+In this guide, we'll walk through deploying an RLS, configuring it in Contour, and configuring an `HTTPProxy` to use it for rate limiting.
+
+**NOTE: you should not consider the RLS deployment in this guide to be production-ready.**
+The instructions and example YAML below are intended to be a demonstration of functionality only.
+Each user will have their own unique production requirements for their RLS deployment.
+
+## Prerequisites
+
+This guide assumes that you have:
+
+- A local KinD cluster created using [the Contour guide][2].
+- Contour installed and running in the cluster using the [quick start][3].
+
+## Deploy an RLS
+
+For this guide, we'll deploy the [Envoy rate limit service][4] as our RLS.
```

Comment:
Maybe just give a couple more sentences on how the RLS works just so it's clear what is happening? (i.e. The RLS uses a Redis cache internally. Blah blah?)

Reply:
copied an excerpt from the Envoy RLS README to give some more color.